{{ cookiecutter.app_name }}/tests/{{ cookiecutter.module_name }}.py

```diff
@@ -0,0 +1,90 @@
+{%- if cookiecutter.test_framework == 'pytest' -%}
+import os
+import sys
+import tempfile
+from pathlib import Path
+
+import pytest
+
+
+def test_suite():
```

Comment:
If this function begins with the word `test`, and the `module_name` also begins or ends with `test`, then pytest will [detect it as a test](https://docs.pytest.org/en/stable/explanation/goodpractices.html#conventions-for-python-test-discovery) and attempt to run it. So this should be renamed, e.g. to `run_tests` as in https://github.com/beeware/toga/pull/1687.

Reply:
Weird - I'm not seeing this manifest when I actually run the tests. However - I can't argue with the general principle, and it's a trivial change to make.